### PR TITLE
Update Addons links in README

### PR DESCRIPTION
**Pull request by pmallol**
Issue:
The Addons links in the README file were linking to a 404 error page.

## What I did
I updated the links to their corresponding github pages.

### CODE REVIEW

**Review by shilman — COMMENTED** [review]
thanks for the fix. can you explain why it's needed tho? the links seem to be fine on github .. is the readme shown on another site?

**Comment by tcp** [comment]
Yes, they're shown in storybook's website and the links are broken. Example: https://storybook.js.org/addons/@storybook/addon-a11y

**Review by ndelangen — APPROVED** [review]

**Review by winkerVSbecks — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -118,23 +118,22 @@ For additional help, join us in the [Storybook Discord](https://discord.gg/story
...
+| [knobs](https://github.com/storybookjs/storybook/tree/next/addons/knobs)                      | Interactively edit component prop data in the Storybook UI                 |
+| [links](https://github.com/storybookjs/storybook/tree/next/addons/links/)                      | Create links between stories                                               |
+| [query params](https://github.com/storybookjs/storybook/tree/next/addons/queryparams/)         | Mock query params                                                          |
+| [storyshots](https://github.com/storybookjs/storybook/tree/next/addons/storyshots/)            | Snapshot testing for components in Storybook                               |
+| [storysource](https://github.com/storybookjs/storybook/tree/next/addons/storysource/)          | View the code of your stories within the Storybook UI                      |
+| [viewport](https://github.com/storybookjs/storybook/tree/next/addons/viewport/)                | Change display sizes and layouts for responsive components using Storybook |
```
Comment (jonniebigodes) [inline README.md:136]: @pmallol checking this it seems that the ones that are leading to 404's are:

 - [design-assets](https://github.com/storybookjs/addon-design-assets)
-  [events](https://github.com/storybookjs/addon-events)
- [cssresources](https://github.com/storybookjs/addon-cssresources)
- [graphql](https://github.com/storybookjs/addon-graphql)
- [queryparams](https://github.com/storybookjs/addon-queryparams)
- [google-analytics](https://github.com/storybookjs/addon-google-analytics)
- [knobs](https://github.com/storybookjs/addon-knobs)

The rest are working as they should. If you could adjust the ones I've mentioned and revert the other ones back as they were we'd appreciate it. It seems that I or @shilman forgot to update the readme when #14831 was addressed and closed.

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@pmallol thank your help in updating the documentation. We really appreciate it. I left one small note regarding this pull request and once it's addressed I'll gladly merge it.

@shilman asides from the 404's that were not updated with #14831,  it seems that the main Storybook readme is still showing in the addons page for some reason, hence why I assume @tcp 's comment.

Stay safe

File: README.md (hunk 1 above)
Comment (shilman) [reply README.md:136]: cc @coderkevin

**Comment by jonniebigodes** [comment]
@pmallol thanks for the time and effort you've put into this. But I'm going to close this down in favor of #15240. The addons are now properly referenced in the README file. 

Looking forward to your next contribution. Stay safe

**Pull request closed without merging**